File: VRBike/wwwroot/js/tourMarker.js
var layerGroup = L.layerGroup();
function addTourMarker(classname, text, iconsize, position, linkurl) {
    console.log('tour marker: ' + text)
    var markersLayer = L.featureGroup();
    var Icon = L.divIcon({
        className: classname,
        html: text,
        iconSize: iconsize,
    });
    var marker = L.marker(position, { icon: Icon })
        .addTo(markersLayer)
    .on("click", function (ev) {
        window.location.href = linkurl;
    });
    layerGroup.addLayer(markersLayer);
    map.addLayer(layerGroup);
}

File: VRBike/Models/TourMarker.cs
namespace VRBike.Models
{
    public class TourMarker
    {
        public int TourMarkerID { get; set; }
        public int TourID { get; set; }
        public string IconSize { get; set; }
        public string Classname { get; set; }
        public string Position { get; set; }
        public string Html { get; set; }
        public string Tooltip { get; set; }
        public string LinkUrl { get; set; }
        public bool Active { get; set; }

    }
}
